packages/playwright-reporter-llm/src/lib/internal/networkProcessing.ts

```
@@ -113,11 +113,49 @@ function deriveNetworkDurationMs(timings: NetworkTimingBreakdown | undefined): n
+}
+
+function parseDatadogIdentifier({
+  value,
+  hexadecimalWidth,
+}: ParseDatadogIdentifierInput): string | undefined {
+  if (value === undefined || !/^\d+$/.test(value)) {
+    return undefined;
+  }
+
+  const identifier = BigInt(value);
+  if (identifier === 0n || identifier > 0xffff_ffff_ffff_ffffn) {
```

ESLint

[error] Invalid group length in numeric value.

(unicorn/numeric-separators-style)